src/schema/associations.ts

```diff
@@ -85,6 +86,16 @@ const associationLookup = {
       sampling_frequency: columns.get('sampling_frequency'),
     }
   },
+  coordsystem: async (file: BIDSFile, options: { maxRows: number }): Promise<{path: string, keys: string[]}> => {
+    const keys = Object.keys(await loadJSON(file, options.maxRows)
+      .catch((e) => {
+        return []
+      }))
```

Comment:
Realized that the await was inside Object.keys. Also, there's no maxrows. We should add a test to make sure the file gets loaded. I think it wouldn't with the extra argument, and nothing to notice the bug.

```suggestion
    const keys = Object.keys(await loadJSON(file).catch((e) => {}))
```

Reply:
FWIW the file *was* getting loaded in my local tests (based on a console.log of the context).

Reply:
Huh. Javascript.